homeassistant/components/goalzero/const.py

```diff
@@ -0,0 +1,58 @@
+"""Constants for the Goal Zero Yeti integration."""
+from datetime import timedelta
+
+from homeassistant.const import (
+    ATTR_VOLTAGE,
+    CONF_BINARY_SENSORS,
+    CONF_NAME,
+    ENERGY_WATT_HOUR,
+    PERCENTAGE,
+    POWER_WATT,
+    TEMP_CELSIUS,
+    TIME_MINUTES,
+    TIME_SECONDS,
+)
+
+DATA_KEY_COORDINATOR = "coordinator"
+DOMAIN = "goalzero"
+DEFAULT_NAME = "Yeti"
+DATA_KEY_API = "api"
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=30)
+
+BINARY_SENSOR_DICT = {
+    "v12PortStatus": ["12V Port Status", CONF_BINARY_SENSORS, None],
```

Comment:
The 2nd value is always `CONF_BINARY_SENSORS`.  If its always going to be the same, please remove it